backend/src/webhook/webhook.controller.ts

```diff
@@ -0,0 +1,86 @@
+import {
+    Controller,
+    Get,
+    Res,
+    Logger,
+    Req, Post,
+} from '@nestjs/common';
+import { ApiUseTags, ApiResponse } from '@nestjs/swagger';
+import { FacebookFeedService } from '../facebook-feed/facebook-feed.service';
+import { ConfigService } from '../config/config.service';
+import { TwitterFeedService } from '../twitter-feed/twitter-feed.service';
+import {YouTubeFeedService} from "../youtube-feed/youtube-feed.service";
+const config = new ConfigService();
+const crypto = require('crypto');
+const SHA = 'sha256=';
+
+@ApiUseTags('webhook')
+@Controller('/api/webhook')
+export class WebhookController {
+    constructor(
+        private readonly facebookFeedService: FacebookFeedService,
+        private readonly twitterFeedService: TwitterFeedService,
+        private readonly youtubeFeedService: YouTubeFeedService,
+        private readonly logger: Logger) {
+        this.logger = new Logger('WebhookController');
+    }
+
+    @Get('facebook')
+    @ApiResponse({status: 200, description: 'Verification endpoint for Facebook fetch'})
```

Comment:
Description for @ApiResponse needs to be about the response value which this particular method and status type is returning. It needs to be updated accordingly. Applicable to all of them in this file.